Source/WebKit/Shared/API/glib/WebKitContextMenuItem.cpp

```diff
@@ -333,6 +333,28 @@ GAction* webkit_context_menu_item_get_gaction(WebKitContextMenuItem* item)
 #endif // ENABLE(CONTEXT_MENUS)
 }
 
+/**
+ * webkit_context_menu_item_get_title:
+ * @item: a #WebKitContextMenuItem
+ *
+ * Gets the title associated to @item.
+ *
+ * Returns: (transfer none): the title associated to the #WebKitContextMenuItem,
+ *    or %NULL if @item is a separator.
+ *
+ * Since: 2.18
```

Comment:
The next stable series will be `2.48`, I guess the `1` there is a typo:

```suggestion
 * Since: 2.48
```